src/ray/raylet/worker_pool_test.cc

```diff
@@ -1149,7 +1147,9 @@ TEST_F(WorkerPoolTest, DeleteWorkerPushPop) {
 TEST_F(WorkerPoolTest, NoPopOnCrashedWorkerProcess) {
   // Start a Java worker process.
   PopWorkerStatus status;
-  auto [proc, token] = worker_pool_->StartWorkerProcess(
+  auto [proc1, token1] = worker_pool_->StartWorkerProcess(
+      Language::JAVA, rpc::WorkerType::WORKER, JOB_ID, &status);
+  auto [proc2, token2] = worker_pool_->StartWorkerProcess(
```

Comment:
This test case can be deleted now. Its only purpose is to test the multi-worker feature. See https://github.com/ray-project/ray/pull/13793.

Reply:
Removed.